### ISSUE TITLE
Unable to cast object of type 'LiveChartsCore.SkiaSharpView.UWP.MotionCanvas' to type 'Windows.UI.Xaml.IUIElement10'

### ISSUE THREAD

**Issue by CarteKiwi** [beto-rodriguez/LiveCharts2#355]
**Describe the bug**
Unable to cast object of type 'LiveChartsCore.SkiaSharpView.UWP.MotionCanvas' to type 'Windows.UI.Xaml.IUIElement10'

**To Reproduce**
1. Execute the Uwp Sample app or create a CartesianChart

```
 <lvc:CartesianChart x:Name="Chart"
    Series="{Binding Series, Mode=OneWay}"
    XAxes="{Binding XAxes, Mode=OneTime}" 
    YAxes="{Binding YAxes, Mode=OneTime}" 
    HorizontalAlignment="Center" VerticalAlignment="Bottom"
    Height="300" Width="850"></lvc:CartesianChart>
```
2. Click on "Events/Cartesian" page
3. See error

**Expected behavior**
No error.

**Screenshots**
![image](https://user-images.githubusercontent.com/12309245/148280884-95df30aa-cab2-447b-b437-33de917876cc.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 (17763)
 - Version 2.0.0-beta.80

**Additional context**
Tested on Windows 10 Iot Core in Debug mode on a Dragonboard 410C.

**Comment by CarteKiwi** [comment]
After investigation, error occurs here:

![image](https://user-images.githubusercontent.com/12309245/148282192-f9a91bc7-e86e-4398-93aa-7add18a54550.png)

I guess the XamlRoot.RasterizationScale is not supported on Windows 17763.

[XamlRoot class](https://docs.microsoft.com/sk-sk/uwp/api/windows.ui.xaml.xamlroot?view=winrt-18362) was introduced in 18362.

![image](https://user-images.githubusercontent.com/12309245/148281831-c1953145-a5bc-4bb1-aaf2-9a68b20e4e9a.png)

**Comment by CarteKiwi** [comment]
@jnayee Hi :) You changed that line in a previous build:

![image](https://user-images.githubusercontent.com/12309245/148306620-1eff69a4-81cd-480f-be33-fc59d269c415.png)

**Comment by jnayee** [comment]
Hi @CarteKiwi!
This line was updated during [#181](https://github.com/beto-rodriguez/LiveCharts2/pull/181), although I only adjusted this particular line to align this with WinUI, where after adjusting and testing on my PC I found achieved the same result, I didn't realise it created a compatibility issue. This was just to try keep code common where possible. I tested reverting this back with project min version set to build 17763 and this built successfully. PR #358 has been raised to correct :-)
